tox.ini

```diff
@@ -19,19 +19,17 @@ whitelist_externals =
     mkdir
     rm
 deps =
-    django31: Django>=3.1,<3.2
-    django32: Django>=3.2,<4.0
+    django42: Django>=4.2,<4.3
     -r{toxinidir}/requirements/test.txt
 passenv = 
     BOKCHOY_HEADLESS
 setenv =
     DJANGO_SETTINGS_MODULE = workbench.settings
     SCREENSHOT_DIR={toxinidir}/var/logs
     SELENIUM_DRIVER_LOG_DIR={toxinidir}/var/logs
-commands = 
+commands =
     mkdir -p {toxinidir}/var/logs
     rm -rf {toxinidir}/var/logs/*
-    pytest {posargs}
```

Comment:
Is it intentional?

Reply:
@awais786 make this change when fixing test cases.